data/test-segmentio.sh

```diff
@@ -2,21 +2,68 @@
 accessKey=$1
 
 # normal case
-curl -i -X POST http://localhost:7070/webhooks/segmentio.json?accessKey=$accessKey \
--H "Content-Type: application/json" \
--d '{
-  "version"   : 1,
-  "type"      : "identify",
-  "userId"    : "019mr8mf4r",
-  "traits"    : {
-      "email"            : "achilles@segment.com",
-      "name"             : "Achilles",
-      "subscriptionPlan" : "Premium",
-      "friendCount"      : 29
-  },
-  "timestamp" : "2012-12-02T00:30:08.276Z"
-}' \
--w %{time_total}
+curl -H "Accept: application/json; version=2.0" \
```

Comment:
this call will return generated for test object specified type. at this case it will be `identify`